### DISCUSSION TITLE
Allow "context menu" key to be rebound

### DISCUSSION THREAD

**Discussion by lualex403** [ppy/osu#13183] [Ideas]
**Describe the bug:** 
Application (Windows Context Menu Key or "menu" key) doesn't work when I try to bind it as a keybind. I have tried other keys, but it's just "apps" key that is unable to be bound to a shortcut.

**osu!lazer version: 2021.226.0**

**Comment by peppy** [comment]
Before we even consider supporting this, can you please show other games which support this?

**Comment by lualex403** [comment]
Im not sure what other games have this feature, I only play csgo and osu. In csgo its not really needed to bind the apps key, but in osu I want to bind it as the toggle extended chat because for my keyboard, they key rests just below my right thumb so it's convenient to press while not interfering with osu's z/x because sometimes when tapping I accidentally hit keys, and my keyboard doesn't have dedicated function keys so I'm only left with (on the bottom row from left to right), "menu, fn (which can't be bound to), and ctrl (uncomfortably far)".

**Comment by peppy** [comment]
Can you try binding in CS:GO? Is it *supported*? As this is a system level key, I am not sure we actually want the ability to override it, so I'm very curious if you can find another game where this is actually a thing.

I'd wager that it isn't.

**Comment by lualex403** [comment]
I tried in csgo, and you can't bind it. Maybe its probably best not to override a system level key.

**Comment by Aegiscarr** [comment]
> Im not sure what other games have this feature

> find another game where this is actually a thing.

I just checked and Minecraft is one of the games I have that let you use the menu key as a keybind. (it also lets you bind the windows key) Probably an anomaly though, can't imagine more modern games having system level keys as an option.
Just thought I'd throw this in here
